src/ansys/sherlock/core/layer.py

```diff
@@ -655,6 +684,8 @@ def update_test_fixtures_by_file(
                 LOG.error(error)
             raise e
 
+    #  First PySherlock Release "0.7.0"
+    @require_version(251)

```

Comment:
Should all of the methods with @require_version have a comment for "Available Since"?

Reply:
waiting till release meeting to know what the releases will be called to know what this should be.